test/prepare.sh

```diff
@@ -2,18 +2,52 @@
 
 set -eux -o pipefail
 
-IMG="${IMG:-controller:test}"
+IMG="${IMG:-localhost/controller:test}"
 LOGDIR="${LOGDIR:-/tmp/logs}"
+CONTAINER_RUNTIME="${CONTAINER_RUNTIME:-}"
+CERT_MANAGER_VERSION="${CERT_MANAGER_VERSION:-1.16.1}"
 
 mkdir -p "${LOGDIR}"
 
+if [ -z "${CONTAINER_RUNTIME}" ]; then
+    if podman version > /dev/null 2>&1;  then
```

Comment:
```suggestion
    if command -v podman &> /dev/null;  then
```
Unless you specifically want to run `podman version` for some reason, suggested way avoids executing heavier podman executable, and command is more standard way of checking for tools. Also, `&>` is more efficient way of redirecting everything.